app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,84 @@
-.check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
```

Comment:
I don't think this is relevant any more. It would still be good to include a comment description of the widths we support in the scss file though - I expect the comment in the html is more likely to get removed at some point.